android/lib/src/main/java/com/ern/api/impl/core/LaunchConfig.java

```diff
@@ -75,6 +75,11 @@
      */
     boolean startRootInOnCreate = true;
 
+    /***
+     * Set this to to true if the root page up click is handled by React native.
+     */
+    boolean rootBackPressHandledByRN = false;
```

Comment:
`false` is default for boolean, you can remove it. Also, for more consistency with the other members in this class (except `startRootInOnCreate`), as well as with the Android framework, can you rename this new member variable to `mRootBackPressHandledByRN`?